src/consumption.cpp

```diff
@@ -710,6 +713,33 @@ bool player::eat( item &food, bool force )
         } else {
             add_msg_player_or_npc( _( "You eat your %s." ), _( "<npcname> eats a %s." ),
                                    food.tname().c_str() );
+            if ( !spoiled && !food.has_flag( "ALLERGEN_JUNK" ) ) {
+                bool has_table_nearby = false;
+                for ( const tripoint &pt : g->m.points_in_radius( pos(), 2 ) ) {
```

Comment:
```suggestion
                for ( const tripoint &pt : g->m.points_in_radius( pos(), 1 ) ) {
```

Sitting two tiles from the table?

Reply:
Yeah, forgot about that.

Reply:
Oh, I just thought radius of 1 is just one tile below the player.
Thanks.